src/js/structs/__tests__/ServiceTree-test.js

```diff
@@ -147,26 +186,135 @@ describe('ServiceTree', function () {
     });
   });
 
+  describe('#filterItemsByFilter', function () {
+
+    beforeEach(function () {
+      this.instance = new ServiceTree({
```

Comment:
yes one option would be to create the `this.instance` and bring everything that is similar into the top-most describe.

then in the individual describes, you can modify `this.instance` in the way you need to for those specific tests, which will cut down on the code.

in another PR sounds good to me. just be sure to do it soon, since I believe that more servicetree tests will be coming